src/global/signal_handler.cc

```diff
@@ -152,6 +158,29 @@ void install_standard_sighandlers(void)
 #include "common/Thread.h"
 #include <errno.h>
 
+void getNameByPid(pid_t pid, char *signal_name) {
+  char proc_pid_path[BUF_SIZE];
+  char buf[BUF_SIZE];
+  sprintf(proc_pid_path, "/proc/%d/status", pid);
```

Comment:
@liewegas : Sir,I think cmdline sometimes will be not clearly to know the process name.If use status I can clearly  know the the thread name as below.

[root@client101 ceph]# cat /proc/11872/cmdline 
/usr/bin/ceph-osd-i0--pid-file/var/run/ceph/osd.0.pid-c/etc/ceph/ceph.conf--clusterceph-f
[root@client101 ceph]# cat /proc/11872/status 
Name:   ceph-osd
State:  S (sleeping)